tools/ubergen/bin/ubergen.ts

```diff
@@ -260,7 +268,30 @@ async function transformPackage(
 ) {
   await fs.mkdirp(destination);
 
-  await copyOrTransformFiles(library.root, destination, allLibraries, uberPackageJson);
+  if (uberPackageJson.ubergen?.stripExperimental && library.packageJson.stability === 'experimental') {
+    // in stripExperimental is on want to only add the L1s of experimental modules.
+    let cfnScopes = library.packageJson['cdk-build'].cloudformation;
+
+    if (cfnScopes === undefined) {
+      return false;
+    }
+
+    if (typeof cfnScopes === 'string') {
+      cfnScopes = [cfnScopes];
+    }
+    const destinationLib = path.join(destination, 'lib');
+    await fs.mkdirp(destinationLib);
+    await cfn2ts(cfnScopes, destinationLib);
```

Comment:
nit: a little js trick

```suggestion
    const destinationLib = path.join(destination, 'lib');
    await fs.mkdirp(destinationLib);
    await cfn2ts([...cfnScopes], destinationLib);
```

Reply:
crap, nvm then. I tried to be fancy and failed.